day14/Python/Product.py

```diff
@@ -0,0 +1,28 @@
+"""
+@author: NormalVad
+@date: 25/10/2020
+"""
+
+def product(a,b):
```

Comment:
There was another part to the question where you needed to calculate the sum of the digits of a number using recursion. Do include the solution to that in this Python script as well.